proto/babylon/incentive/genesis.proto

```diff
@@ -141,4 +141,6 @@ enum StakeholderType {
   FINALITY_PROVIDER = 0;
   // BTC staker stakeholder type
   BTC_STAKER = 1;
+  // Coostaker stakeholder type
+  COOSTAKER = 2;
```

Comment:
new stakeholder type to be able to see how many rewards are comming from `x/coostaking`